### PR TITLE
[DO NOT MERGE] Uses the updated version for bls12377js with the new greatest bit location

### PR DESCRIPTION
**Pull request by kobigurk**
## Description

https://github.com/celo-org/bls-zexe/pull/140 changes the location of the greatest bit, so we must update bls12377js as well.

### CODE REVIEW

**Comment by gakonst** [comment]
This is the relevant change in the underlying library: https://github.com/celo-org/bls12377js/commit/ca0092230efcc5e6c190689d564d8c61b5ba710e.

File: yarn.lock [hunk 1]
```diff
@@ -10533,6 +10533,18 @@ bls12377js@celo-org/bls12377js#566137fd6b9568ef105a3aeaf8e36ff03da91d49:
...
+  dependencies:
+    "@stablelib/blake2xs" "0.10.4"
+    "@types/node" "^12.11.7"
+    big-integer "^1.6.44"
+    chai "^4.2.0"
+    mocha "^6.2.2"
+    ts-node "^8.4.1"
+    typescript "^3.6.4"
+
 "bls12377js@https://github.com/celo-org/bls12377js#400bcaeec9e7620b040bfad833268f5289699cac":
```
Comment (jmrossy) [inline yarn.lock:10548]: If this version is still here, it implies some package somewhere still relies on that version. Is that an issue?

Comment (kobigurk) [reply yarn.lock:10548]: I believe it's old versions of utils that depend on it, and some packages depend on old versions of utils.

File: .circleci/config.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ e2e-defaults: &e2e-defaults
   docker:
     - image: celohq/circleci:geth1.9-2
   environment:
-    CELO_BLOCKCHAIN_BRANCH_TO_TEST: master
+    CELO_BLOCKCHAIN_BRANCH_TO_TEST: kobigurk/zexe-greatest-bit
```
Comment (jmrossy) [inline .circleci/config.yml:39]: Please revert when you're done :)

**Review by jmrossy — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (kobigurk) [reply .circleci/config.yml:39]: Yep :)

**Comment by aaronmgdr** [comment]
close?

**Comment by barbaraliau** [comment]
Closing as this hasn't had movement for several months. Please reopen if this is still relevant!

**Pull request closed without merging**